enforcer/datapath_tcp.go

```diff
@@ -71,12 +71,10 @@ func (d *Datapath) processNetworkTCPPackets(p *packet.Packet) (err error) {
 	conn.Lock()
 	defer conn.Unlock()
 
-	d.netTCP.IncomingPackets++
```

Comment:
I'm not in favor of removing these stats. These are per datapath stats so not that much memory and definitely not that much compute overhead.